torch_geometric/nn/models/gnn_explainer.py

```diff
@@ -125,13 +155,21 @@ def __subgraph__(self, node_idx, x, edge_index, **kwargs):
                 item = item[edge_mask]
             kwargs[key] = item
 
-        return x, edge_index, mapping, edge_mask, kwargs
+        return x, edge_index, mapping, edge_mask, subset, kwargs
 
     def __loss__(self, node_idx, log_logits, pred_label):
         # node_idx is -1 for explaining graphs
-        loss = -log_logits[
-            node_idx, pred_label[node_idx]] if node_idx != -1 else -log_logits[
-                0, pred_label[0]]
+        if self.return_type == 'regression':
+            if node_idx != -1:
+                loss = torch.cdist(
```

Comment:
Does this work better in practice compared to something like `MSELoss`?

Reply:
In terms of performance, differences between both seem to be insignificant. But yes, if that looks better, we can also use something like
```
self.mse_loss = torch.nn.MSELoss(reduction='sum')
...
loss = torch.sqrt(self.mse_loss(log_logits[node_idx], pred_label[node_idx]))
```
What do you think?